src/Units/KaijuWarsUnits.java

```diff
@@ -0,0 +1,783 @@
+    turkeyAir.turkeyLand = turkeyLand;
+    turkeyAir.turkeyEgg = turkeyEgg;
+    kaijuModels.add(turkeyAir);
+    extras.add(turkeyLand);
+    extras.add(turkeyEgg);
+    final KaijuWarsKaiju.BigDonkRampage rampager = new KaijuWarsKaiju.BigDonkRampage();
+    extras.add(rampager);
+    kaijuModels.add(new KaijuWarsKaiju.BigDonk(rampager));
+    final KaijuWarsKaiju.SnekTunneler tunneler = new KaijuWarsKaiju.SnekTunneler();
+    extras.add(tunneler);
+    kaijuModels.add(new KaijuWarsKaiju.Snek(tunneler));
+    final KaijuWarsKaiju.UFOAbducts abductor = new KaijuWarsKaiju.UFOAbducts();
+    extras.add(abductor);
+    kaijuModels.add(new KaijuWarsKaiju.UFO(abductor));
+
+    // Inscribe those war machines obtainable from an Airport.
+    airportModels.add(new Fighter());
+    airportModels.add(new Bomber());
+    airportModels.add(new Helicopter());
+    airportModels.add(new Bushplane());
+
+    // Carrier gets to build all normal air units
+    UnitModel carrier = new SkyCarrier();
+    for (UnitModel um : airportModels)
+      carrier.baseActions.add(1, new UnitProduceLifecycle.UnitProduceFactory(um));
+
+    airportModels.add(new BigBoy());
+    airportModels.add(new GuncrossWing());
+    airportModels.add(new Kaputnik());
+    extras.add(new GuncrossRobot());
+    airportModels.add(carrier);
+
+    // Dump these lists into a hashmap for easy reference later.
+    kjwModels.shoppingList.put(TerrainType.FACTORY, factoryModels);
+    kjwModels.shoppingList.put(TerrainType.SEAPORT, kaijuModels);
+    kjwModels.shoppingList.put(TerrainType.AIRPORT, airportModels);
+
+    // Compile one master list of everything we can build.
+    for (UnitModel um : factoryModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : airportModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : kaijuModels)
+      kjwModels.unitModels.add(um);
+    for (UnitModel um : extras)
+      kjwModels.unitModels.add(um);
+
+    // Multiplies all movement (and ranges) by N
+    for( UnitModel um : kjwModels.unitModels )
+      um.baseMovePower *= KaijuWarsWeapons.KAIJU_SCALE_FACTOR;
+
+    return kjwModels;
+  }
+
+  public void registerStateTrackers(GameInstance gi)
+  {
+    super.registerStateTrackers(gi);
+
+    StateTracker.instance(gi, UnitTurnPositionTracker.class);
+
+    UnitResurrectionTracker rezzer = StateTracker.instance(gi, UnitResurrectionTracker.class);
+    // Populate resurrection pairs
+    GameReadyModels grms = gi.rules.unitModelScheme.getGameReadyModels();
+    for( UnitModel um : grms.unitModels )
+    {
+      KaijuWarsUnitModel umCast = (KaijuWarsUnitModel) um;
+      if( null == umCast.resurrectsAs )
+        continue;
+
+      // We have a type that resurrects; find the type it rezzes to
+      for( UnitModel rezToModel : grms.unitModels )
+      {
+        if( !rezToModel.name.equals(umCast.resurrectsAs) )
+          continue;
+        rezzer.resurrectionTypeMap.put(um, rezToModel);
+        break;
+      }
```

Comment:
Why make `resurrectAs` a `String` instead of a `UnitModel`? Avoid this loop and might streamline other things too.

Reply:
I was worried about whether that would work after save/load.